person.rb

```diff
@@ -2,7 +2,7 @@ class Person
   attr_accessor :name, :age
   attr_reader :id
 
-  def initialize(name: 'Unknown', age: nil, parent_permission: true)
+  def initialize(name = 'Unknown', age = nil, parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
```

Comment:
- [ ] Great job so far @MarcoDDM There are just a few changes I will suggest you implement and you'll be on your way to the next Milestone 😉 
- [ ] In your `Person.rb` class, I noticed that you didn't implement the Project Requirement that stated that you need to 

> Turn your Person class to Nameable
> Make sure that your Person class inherits from Nameable

- However, that hasn't been implemented in your `Person.rb` class so kindly include this as well so all the project requirements can be met as expected 😉